voluncheer/voluncheer/settings.py

```diff
@@ -168,9 +168,9 @@
 EMAIL_BACKEND = "django_ses.SESBackend"
 AWS_ACCESS_KEY_ID = os.getenv("AWS_ACCESS_KEY")
 AWS_SECRET_ACCESS_KEY = os.getenv("AWS_SECRET_ACCESS_KEY")
-AWS_SES_REGION_NAME = "us-west-2"  # (ex: us-east-2)
+AWS_SES_REGION_NAME = "us-east-1"  # (ex: us-east-2)
```

Comment:
Sadly, `us-east-1` doesn't have SES. I believe we have to use `us-east-2` for SES specifically. Also, please remove the comments that follow these lines, they don't really add anything.